style.css

```diff
@@ -0,0 +1,160 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  background-attachment: fixed;
+  background-position: center;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/60164239/129299376-f5ad6782-1d57-43af-bfe4-d08aac11811c.png)

- [ ]  You should use the right font family for your project, please add it, when you click on any element in Figma you can click in inspect as well to see what is the typography.
- [ ] Add padding to fix the position of your elements as you can see in the blue color.